Description: Purges native windows integration for slave from Jenkins.
 Although this may be useful in some deployments the requirements in terms
 of additional packages are not insignificant and contain a number of forks.
Author: James Page <james.page@ubuntu.com>
Forwarded: not-needed

Index: jenkins/core/src/main/java/hudson/slaves/SlaveComputer.java
===================================================================
--- jenkins.orig/core/src/main/java/hudson/slaves/SlaveComputer.java	2012-07-24 16:32:51.254005727 +0100
+++ jenkins/core/src/main/java/hudson/slaves/SlaveComputer.java	2012-07-24 16:33:07.914065141 +0100
@@ -35,7 +35,6 @@
 import hudson.util.RingBufferLogHandler;
 import hudson.util.Futures;
 import hudson.FilePath;
-import hudson.lifecycle.WindowsSlaveInstaller;
 import hudson.Util;
 import hudson.AbortException;
 import hudson.remoting.Launcher;
@@ -365,7 +364,6 @@
         channel.pinClassLoader(getClass().getClassLoader());
 
         channel.call(new SlaveInitializer());
-        channel.call(new WindowsSlaveInstaller(remoteFs));
         for (ComputerListener cl : ComputerListener.all())
             cl.preOnline(this,channel,root,taskListener);
 
